src/blocks/BulletPointBlock/BulletPointBlock.tsx

```diff
@@ -0,0 +1,217 @@
+import styled from '@emotion/styled'
+import React from 'react'
+import {
+  ContentWrapper,
+  getMinimalColorStyles,
+  LAPTOP_BP_UP,
+  MOBILE_BP_DOWN,
+  MOBILE_BP_UP,
+  SectionWrapper,
+  TABLET_BP_UP,
+} from 'components/blockHelpers'
+import { DeferredImage } from 'components/DeferredImage'
+import { getStoryblokImage, Image } from 'utils/storyblok'
+import {
+  BrandPivotBaseBlockProps,
+  MarkdownHtmlComponent,
+  MinimalColorComponent,
+} from 'blocks/BaseBlockProps'
+import { FontSizes, Heading } from 'components/Heading/Heading'
+
+const BulletPointSectionWrapper = styled(SectionWrapper)`
+  overflow-x: hidden;
+`
+
+const InnerWrapper = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  min-width: 100%;
+
+  ${MOBILE_BP_DOWN} {
+    padding-left: 8px;
+    padding-right: 8px;
+  }
+
+  ${MOBILE_BP_UP} {
+    margin-left: -1.5rem;
+  }
+
+  ${LAPTOP_BP_UP} {
+    margin-left: -3.5rem;
+  }
+`
+
+const BulletPointTitle = styled(Heading)`
+  margin-bottom: 1rem;
+`
+
+const BulletPoint = styled.div<{
+  alignCenter: boolean
+  bulletPointLayout: boolean
+  columns: number
+}>`
+  display: flex;
+  flex-direction: ${(props) => (props.bulletPointLayout ? 'row' : 'column')};
```

Comment:
[Nit/personal preference]: Maybe destructure the props that are used in the `styled` components? To me that's more readable :)
```suggestion
  flex-direction: ${({ bulletPointLayout }) => (bulletPointLayout ? 'row' : 'column')};
```

Reply:
I think I'm so used to writing `props` that I find it easier to read :D

Reply:
Oh, OK. It's just so much more... characters that it gets blurry to me. 

Anyway, this might be something for the code style guide, since for example my impulse is always to refactor to destructuring syntax (did it just a few days ago in the `Button` component in `web-onboarding`) and it's probably a waste of time if we'd keep refactoring back and forth haha 😄

Reply:
Agreed! I could happily use that pattern, so as you say, we could commit to one or the other 🙌